src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -97,14 +114,16 @@ public void shouldBeAbleToCheckinItemWithoutLoan() {
 
     IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
     UUID servicePointId = servicePointsFixture.cd1().getId();
-    DateTime checkInDate = DateTime.now();
 
-    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    loansFixture.checkInByBarcode(item, checkInDateTime, servicePointId);
 
     JsonObject lastCheckIn = itemsClient.get(item.getId()).getJson()
       .getJsonObject("lastCheckIn");
 
-    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    DateTime actualCheckinDateTime = JsonPropertyFetcher
+      .getDateTimeProperty(lastCheckIn, "dateTime");
+
+    assertThat(actualCheckinDateTime, is(checkInDateTime));
```

Comment:
I thought the behaviour was that this was mean to always be the date time when the check in was processed on the system? Or is this the same, because coincidentally the check in date time and the processed / system date time are the same?

Reply:
They are the same because they are all taken from the clockmanager, which is mocked in tests.

Reply:
What are these tests trying to demonstrate?

As I understand it, the valuable behaviour is that the last check in date time is always the date time when the check in was processed on the system. Is that the case?

Reply:
It shows the presence of the last checkin object in item after item checkin without active loan(in-house use) and that all the required fields are set correctly. What do you think these tests should be demonstrating?

Reply:
I've updated the tests to show that the correct time is set to the last checkin object regardless of the date passed to the checkin.

Reply:
Thank you